src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  let total = days * dayPrice;
```

Comment:
Consider using `const` for the `total` variable since it's not being reassigned but only modified in the if-else blocks. This adheres to the checklist item 3, preferring `const` over `let` where possible to avoid unintentional changes.

Reply:
This error occurs when const with total is used: 
<img width="645" alt="Screenshot 5784-12-03 at 12 44 50 PM" src="https://github.com/user-attachments/assets/1dca0fab-f296-474c-a696-4d37dac84df8">